manager/config/prod/deployment_configmap.yaml

```diff
@@ -9,24 +9,31 @@ data:
   SECRET_PROVIDER_URL: "http://secret-provider.m4d-system.svc.cluster.local:5555/get-secret"
   SECRET_PROVIDER_ROLE: "demo"
   #catalog connector
-  CATALOG_CONNECTOR_URL: "egr-connector:50084"
-  CATALOG_PROVIDER_NAME: "EGERIA"
+  # CATALOG_CONNECTOR_URL: "egr-connector:50084"
+  # CATALOG_PROVIDER_NAME: "EGERIA"
+  CATALOG_CONNECTOR_URL: "wkc-connector:50090"
+  CATALOG_PROVIDER_NAME: "WKC"
```

Comment:
why?

Reply:
We used this branch for the demo on Oct 15. Hence we had to make the default connectors to point to WKC.

Reply:
We are not changing the prod config just because of a demo